src/demo/app/components/shortcut-service-demo.component.html

```diff
@@ -27,7 +27,7 @@ <h4>
                 <div class="form-group row">
                     <select
                         style="margin-left: 15px;"
-                        class="form-control col-sm-2"
+                        class="form-control col-sm-3"
                         name="key1selector"
                         formControlName="key1"
                     >
```

Comment:
Can we also add a placeholder for the Select list.  Seems weird to have one for the text box.  Maybe just "first key".  Here is a link to how to do a place holder: https://stackoverflow.com/a/5859221/5997923